### PR TITLE
Don't remove existing uses if they are the same as the created string for use statements

### PR DESCRIPTION
**Pull request by junichi11**
- https://github.com/apache/netbeans/pull/5681
- Avoid changing the file state if the created string is the same as the existing use statements

### CODE REVIEW

**Comment by junichi11** [comment]
I didn't fix the "multiple default namespaces" issue because I didn't think it's important.
I hope that the regressions are resolved in NB19.

**Comment by tmysik** [comment]
@junichi11 I agree, it makes sense to me. We can always wait for a new report.

File: php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java [hunk 1]
```diff
@@ -493,13 +505,22 @@ private String createStringForCommonUse(List<UsePart> useParts) {
         return result.toString();
     }
 
-    private void processExistingUses() {
-        ExistingUseStatementVisitor visitor = new ExistingUseStatementVisitor();
-        Program program = parserResult.getProgram();
-        if (program != null) {
-            program.accept(visitor);
+    private String getExistingUses(List<OffsetRange> usedRanges) {
+        String existingUses = ""; // NOI18N
```
Comment (tmysik) [inline php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java:509]: Nitpick:

```suggestion
        String existingUses = EMPTY_STRING;
```

**Review by tmysik — APPROVED** [review]

File: php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java (hunk 1 above)
Comment (junichi11) [reply php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java:509]: Oops, thank you for catching it :)

Comment (junichi11) [reply php/php.editor/src/org/netbeans/modules/php/editor/actions/FixUsesPerformer.java]: Fixed.

**Pull request merged**